.github/workflows/ci.yml

```diff
@@ -39,4 +37,169 @@ jobs:
+    - name: Python (CPP) Wrapper Code
+      shell: bash -el {0}
+      run: |
+            conda activate impalib
+            cd src/impa
+            python3 main_tsp.py --filteringFlag=True --alpha=0.5 --augmFlag=True --threshold=-0.0001 --nITER=200 --inputPath=inputs_1000_nNodes_random --testFile=0 --lkhSolFlag=True --maxAugmCount=10
+            python3 main_tsp.py --nNodes=10 --filteringFlag=True --alpha=0.5 --augmFlag=True --threshold=-0.0001 --nITER=200 --randomTestFlag=True --lkhSolFlag=True --maxAugmCount=20
+            
+  ksat-demo-job: 
+    name: Ksat Job
+    runs-on: ${{ matrix.os }}
+    strategy: 
+      matrix:
+        os: [macos-latest, ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-python@v5
+      with: 
+        python-version: '3.9'
+    - uses : conda-incubator/setup-miniconda@v3
+      with:
+        auto-update-conda: true
+        activate-environment: impalib
+        python-version: '3.9'
+    - name: CPP Header-Only Code
+      shell: bash -el {0}
+      run: |
+            conda activate impalib
+            python3 -m pip install --user .
+            cd examples/Ksat
+            cmake -B build && cmake --build build && cd build && ./demo
+    - name: Pure Python Code
+      shell: bash -el {0}
+      run: |
+            conda activate impalib
+            cd test/python_ksat/src
+            python3 main_ksat.py --filteringFlag=True --threshold=-0.0001 --nITER=200 --alpha=0.5 --randomTestFlag=True --nConstraints=20 --nVariables=10 --kVariable=3
+    - name: Python (CPP) Wrapper Code
+      shell: bash -el {0}
+      run: |
+            conda activate impalib
+            cd src/impa
+            python3 main_ksat.py --filteringFlag=True --threshold=-0.0001 --nITER=200 --inputPath=inputs_ksat_zeromean_var_10_samples_500 --outputPath=outputs_ksat_zeromean_var_10_samples_500 --PPElements=2 --testFile=112 --alpha=0.5 --var=8 --typeMetrics=2 --overwrite=1 --PPFlag=True
+            python3 main_ksat.py --filteringFlag=True --threshold=-0.0001 --nITER=200 --PPElements=2 --alpha=0.5 --var=8 --randomTestFlag=True --nConstraints=200 --nVariables=90 --kVariable=5 --PPFlag=True
+
+  kcmwm-demo-job:
+    name: Kcmwm Job
+    runs-on: ${{ matrix.os }}
+    strategy: 
+      matrix:
+        os: [macos-latest, ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-python@v5
+      with: 
+        python-version: '3.9'
+    - uses: conda-incubator/setup-miniconda@v3
+      with:
+        auto-update-conda: true
+        activate-environment: impalib
+        python-version: '3.9'
+    - name: CPP Header-Only Code
+      shell: bash -el {0}
+      run: |
+            conda activate impalib
+            python3 -m pip install --user .
+            cd examples/KcMwm
+            cmake -B build && cmake --build build && cd build && ./demo
+            conda deactivate
```

Comment:
"conda deactivate" appears only here

Reply:
This should be addressed in [`30a9d11e24eb08e93274823c4d3e509a90e4d319`](https://github.com/RustomAlexios/IMPALIB/pull/14/commits/30a9d11e24eb08e93274823c4d3e509a90e4d319)
Thank you!